src/tests/regress/README.md

```diff
@@ -45,7 +45,7 @@ RUST_BACKTRACE=1 target/debug/risingwave_regress_test -h 127.0.0.1 \
 * Run tests against PostgreSQL. Make sure PostgreSQL is running.
 
 ```shell
-RUST_BACKTRACE=1 target/debug/risingwave_regress_test -h 127.0.0.1 \
+RUST_BACKTRACE=1 target/debug/risingwave_regress_test --host 127.0.0.1 \
```

Comment:
cc @arkbriar

Reply:
Thanks for the kind reminder. It seems it won't affect the operator.